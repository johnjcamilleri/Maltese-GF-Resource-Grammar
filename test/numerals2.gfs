-- Treebank numerals test (>1000)
-- Run using the following command:
-- ./_gf < test/numerals2.gfs > test/numerals2.out

se utf8
i NumeralMlt.gf
rf -lines -tree -file=test/numerals2.trees | l -table
