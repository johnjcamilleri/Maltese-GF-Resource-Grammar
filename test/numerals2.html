<html>
<head>
	<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
	<style type="text/css">
		body { font: 16px/1.5 serif; }
		div { margin-left: 2em; }
		#text-output { margin:0; color:#999; }
		#text-output b { display: inline-block; width: 10em; font-weight:normal; }
		#text-output em { font-style: italic; color:#000; }
		#text-output em.correct, .correct { color:green; }
		#text-output em.incorrect, .incorrect { color:red; }
		#text-output em.gold, .gold { background-color:gold; padding:0 5px; }

		table { border-collapse:collapse; border-spacing:0; }
		tr:nth-child(odd) { background-color:#eee; }
		tr.separator { background-color:#000; font-size:2px; }
		td { padding:0.2em 1em; border-width:1px 0; border-style:solid; border-color:#ccc; }
		td:last-child { font-style:italic; }

	</style>
</head>
<body>

<h1>Tree linearisations for: numerals2</h1>
<p><em>Generated Mon, 20 Feb 2012 10:07:14 +0100</em></p>
<p><span class="correct">140 correct</span>, <span class="incorrect">16 incorrect</span></p>
<hr/>

<div id="text-output">
<div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>elf</em></div><div><b>NumAdjectival</b><em>elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- elf</em></div><div><b>NumAdjectival</b><em>l- elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>elfejn</em></div><div><b>NumAdjectival</b><em>elfejn</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- elfejn</em></div><div><b>NumAdjectival</b><em>l- elfejn</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tlett elef</em></div><div><b>NumAdjectival</b><em>tlett elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tlett elef</em></div><div><b>NumAdjectival</b><em>it- tlett elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>disat elef</em></div><div><b>NumAdjectival</b><em>disat elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>id- disat elef</em></div><div><b>NumAdjectival</b><em>id- disat elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>għaxart elef</em></div><div><b>NumAdjectival</b><em>għaxart elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- għaxart elef</em></div><div><b>NumAdjectival</b><em>l- għaxart elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>ħdax-il elf</em></div><div><b>NumAdjectival</b><em>ħdax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- ħdax-il elf</em></div><div><b>NumAdjectival</b><em>il- ħdax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tnax-il elf</em></div><div><b>NumAdjectival</b><em>tnax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tnax-il elf</em></div><div><b>NumAdjectival</b><em>it- tnax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tlettax-il elf</em></div><div><b>NumAdjectival</b><em>tlettax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tlettax-il elf</em></div><div><b>NumAdjectival</b><em>it- tlettax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>dsatax-il elf</em></div><div><b>NumAdjectival</b><em>dsatax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>id- dsatax-il elf</em></div><div><b>NumAdjectival</b><em>id- dsatax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>għoxrin elf</em></div><div><b>NumAdjectival</b><em>għoxrin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- għoxrin elf</em></div><div><b>NumAdjectival</b><em>l- għoxrin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>wieħed u għoxrin elf</em></div><div><b>NumAdjectival</b><em>wieħed u għoxrin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- wieħed u għoxrin elf</em></div><div><b>NumAdjectival</b><em>il- wieħed u għoxrin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tnejn u għoxrin elf</em></div><div><b>NumAdjectival</b><em>tnejn u għoxrin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tnejn u għoxrin elf</em></div><div><b>NumAdjectival</b><em>it- tnejn u għoxrin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tletin elf</em></div><div><b>NumAdjectival</b><em>tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tletin elf</em></div><div><b>NumAdjectival</b><em>it- tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>wieħed u tletin elf</em></div><div><b>NumAdjectival</b><em>wieħed u tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- wieħed u tletin elf</em></div><div><b>NumAdjectival</b><em>il- wieħed u tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tnejn u tletin elf</em></div><div><b>NumAdjectival</b><em>tnejn u tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tnejn u tletin elf</em></div><div><b>NumAdjectival</b><em>it- tnejn u tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>disgħa u disgħin elf</em></div><div><b>NumAdjectival</b><em>disgħa u disgħin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>id- disgħa u disgħin elf</em></div><div><b>NumAdjectival</b><em>id- disgħa u disgħin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitt elf</em></div><div><b>NumAdjectival</b><em>mitt elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitt elf</em></div><div><b>NumAdjectival</b><em>il- mitt elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u wieħed elf</em></div><div><b>NumAdjectival</b><em>mija u wieħed elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u wieħed elf</em></div><div><b>NumAdjectival</b><em>il- mija u wieħed elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u sitt elef</em></div><div><b>NumAdjectival</b><em>mija u sitt elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u sitt elef</em></div><div><b>NumAdjectival</b><em>il- mija u sitt elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u għaxart elef</em></div><div><b>NumAdjectival</b><em>mija u għaxart elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u għaxart elef</em></div><div><b>NumAdjectival</b><em>il- mija u għaxart elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u ħdax-il elf</em></div><div><b>NumAdjectival</b><em>mija u ħdax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u ħdax-il elf</em></div><div><b>NumAdjectival</b><em>il- mija u ħdax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u tletin elf</em></div><div><b>NumAdjectival</b><em>mija u tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u tletin elf</em></div><div><b>NumAdjectival</b><em>il- mija u tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u wieħed u erbgħin elf</em></div><div><b>NumAdjectival</b><em>mija u wieħed u erbgħin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u wieħed u erbgħin elf</em></div><div><b>NumAdjectival</b><em>il- mija u wieħed u erbgħin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u sittin elf</em></div><div><b>NumAdjectival</b><em>mija u sittin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u sittin elf</em></div><div><b>NumAdjectival</b><em>il- mija u sittin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitejn elf</em></div><div><b>NumAdjectival</b><em>mitejn elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitejn elf</em></div><div><b>NumAdjectival</b><em>il- mitejn elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erba' mija u sittax-il elf</em></div><div><b>NumAdjectival</b><em>erba' mija u sittax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- erba' mija u sittax-il elf</em></div><div><b>NumAdjectival</b><em>l- erba' mija u sittax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>ħdax-il elf u tnax</em></div><div><b>NumAdjectival</b><em>ħdax-il elf u tnax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- ħdax-il elf u tnax</em></div><div><b>NumAdjectival</b><em>il- ħdax-il elf u tnax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erbgħin elf</em></div><div><b>NumAdjectival</b><em>erbgħin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- erbgħin elf</em></div><div><b>NumAdjectival</b><em>l- erbgħin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>wieħed u disgħin elf u seba' mija</em></div><div><b>NumAdjectival</b><em>wieħed u disgħin elf u seba' mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- wieħed u disgħin elf u seba' mija</em></div><div><b>NumAdjectival</b><em>il- wieħed u disgħin elf u seba' mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitt elf u mija</em></div><div><b>NumAdjectival</b><em>mitt elf u mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitt elf u mija</em></div><div><b>NumAdjectival</b><em>il- mitt elf u mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitt elf u wieħed u tletin</em></div><div><b>NumAdjectival</b><em>mitt elf u wieħed u tletin</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitt elf u wieħed u tletin</em></div><div><b>NumAdjectival</b><em>il- mitt elf u wieħed u tletin</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u wieħed u għoxrin elf u mija</em></div><div><b>NumAdjectival</b><em>mija u wieħed u għoxrin elf u mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u wieħed u għoxrin elf u mija</em></div><div><b>NumAdjectival</b><em>il- mija u wieħed u għoxrin elf u mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitejn elf u erba' mija u għaxra</em></div><div><b>NumAdjectival</b><em>mitejn elf u erba' mija u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitejn elf u erba' mija u għaxra</em></div><div><b>NumAdjectival</b><em>il- mitejn elf u erba' mija u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>sitt elf</em> &rarr; <em class='gold'>sitt mitt elf</em></div><div><b>NumAdjectival</b><em class='incorrect'>sitt elf</em> &rarr; <em class='gold'>sitt mitt elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>is- sitt elf</em> &rarr; <em class='gold'>is- sitt mitt elf</em></div><div><b>NumAdjectival</b><em class='incorrect'>is- sitt elf</em> &rarr; <em class='gold'>is- sitt mitt elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>sitt mija u għaxart elef u għaxra</em></div><div><b>NumAdjectival</b><em>sitt mija u għaxart elef u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>is- sitt mija u għaxart elef u għaxra</em></div><div><b>NumAdjectival</b><em>is- sitt mija u għaxart elef u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>sebat elf u sitt mija u għaxra</em> &rarr; <em class='gold'>sebat elef u sitt mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>sebat elf u sitt mija u għaxar</em> &rarr; <em class='gold'>sebat elef u sitt mija u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>is- sebat elf u sitt mija u għaxra</em> &rarr; <em class='gold'>is- sebat elef u sitt mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>is- sebat elf u sitt mija u għaxar</em> &rarr; <em class='gold'>is- sebat elef u sitt mija u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tmin mija u ħamsin elf u sbatax</em></div><div><b>NumAdjectival</b><em>tmin mija u ħamsin elf u sbatax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tmin mija u ħamsin elf u sbatax</em></div><div><b>NumAdjectival</b><em>it- tmin mija u ħamsin elf u sbatax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>disat elf u mija u għaxra</em> &rarr; <em class='gold'>disa' mitt elf u mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>disat elf u mija u għaxar</em> &rarr; <em class='gold'>disa' mitt elf u mija u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>id- disat elf u mija u għaxra</em> &rarr; <em class='gold'>id- disa' mitt elf u mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>id- disat elf u mija u għaxar</em> &rarr; <em class='gold'>id- disa' mitt elf u mija u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>disat elf u mitejn</em> &rarr; <em class='gold'>disa' mitt elf u mitejn</em></div><div><b>NumAdjectival</b><em class='incorrect'>disat elf u mitejn</em> &rarr; <em class='gold'>disa' mitt elf u mitejn</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>id- disat elf u mitejn</em> &rarr; <em class='gold'>id- disa' mitt elf u mitejn</em></div><div><b>NumAdjectival</b><em class='incorrect'>id- disat elf u mitejn</em> &rarr; <em class='gold'>id- disa' mitt elf u mitejn</em></div></div></div><hr/>
</div>

</body>
</html>