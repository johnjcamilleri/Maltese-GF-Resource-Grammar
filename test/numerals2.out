=> Starting gf with custom -path flag...
                              
         *  *  *              
      *           *           
    *               *         
   *                          
   *                          
   *        * * * * * *       
   *        *         *       
    *       * * * *  *        
      *     *      *          
         *  *  *              
                              
This is GF version 3.2. 
License: see help -license.   
Bug reports: http://code.google.com/p/grammatical-framework/issues/list


Languages:
> 24 msec
> 4 msec
> 4 msec
> 4 msec
> 4 msec
> + reading /home/john/GF (darcs)/lib/src/abstract/Common.gfo
+ reading /home/john/GF (darcs)/lib/src/abstract/Cat.gfo
+ reading /home/john/GF (darcs)/lib/src/abstract/Numeral.gfo
+ reading /home/john/GF (darcs)/lib/src/prelude/Predef.gfo
+ reading /home/john/GF (darcs)/lib/src/prelude/Prelude.gfo
+ reading /home/john/GF (darcs)/lib/src/common/ParamX.gfo
+ reading /home/john/GF (darcs)/lib/src/common/CommonX.gfo
+ reading /home/john/Documents/GF/Maltese RGL/maltese/PatternsMlt.gfo
+ reading /home/john/Documents/GF/Maltese RGL/maltese/ResMlt.gfo
+ reading /home/john/Documents/GF/Maltese RGL/maltese/CatMlt.gfo
+ reading /home/john/Documents/GF/Maltese RGL/maltese/NumeralMlt.gfo
36 msec
> 4 msec
> s . NCard => NumNominative => wieħed
s . NCard => NumAdjectival => wieħed
s . NOrd => NumNominative => l- ewwel
s . NOrd => NumAdjectival => l- ewwel
n . NumSg
4 msec
> s . NCard => NumNominative => tnejn
s . NCard => NumAdjectival => żewġ
s . NOrd => NumNominative => it- tieni
s . NOrd => NumAdjectival => it- tieni
n . NumDual
8 msec
> s . NCard => NumNominative => tlieta
s . NCard => NumAdjectival => tliet
s . NOrd => NumNominative => it- tielet
s . NOrd => NumAdjectival => it- tielet
n . NumPl
4 msec
> s . NCard => NumNominative => erbgħa
s . NCard => NumAdjectival => erba'
s . NOrd => NumNominative => ir- raba'
s . NOrd => NumAdjectival => ir- raba'
n . NumPl
4 msec
> s . NCard => NumNominative => ħamsa
s . NCard => NumAdjectival => ħames
s . NOrd => NumNominative => il- ħames
s . NOrd => NumAdjectival => il- ħames
n . NumPl
12 msec
> s . NCard => NumNominative => sitta
s . NCard => NumAdjectival => sitt
s . NOrd => NumNominative => is- sitt
s . NOrd => NumAdjectival => is- sitt
n . NumPl
4 msec
> s . NCard => NumNominative => sebgħa
s . NCard => NumAdjectival => seba'
s . NOrd => NumNominative => is- seba'
s . NOrd => NumAdjectival => is- seba'
n . NumPl
8 msec
> s . NCard => NumNominative => tmienja
s . NCard => NumAdjectival => tmin
s . NOrd => NumNominative => it- tmin
s . NOrd => NumAdjectival => it- tmin
n . NumPl
4 msec
> s . NCard => NumNominative => disgħa
s . NCard => NumAdjectival => disa'
s . NOrd => NumNominative => id- disa'
s . NOrd => NumAdjectival => id- disa'
n . NumPl
4 msec
> s . NCard => NumNominative => għaxra
s . NCard => NumAdjectival => għaxar
s . NOrd => NumNominative => l- għaxar
s . NOrd => NumAdjectival => l- għaxar
n . NumPl
4 msec
> s . NCard => NumNominative => ħdax
s . NCard => NumAdjectival => ħdax-il
s . NOrd => NumNominative => il- ħdax-il
s . NOrd => NumAdjectival => il- ħdax-il
n . NumPl
4 msec
> s . NCard => NumNominative => tlettax
s . NCard => NumAdjectival => tlettax-il
s . NOrd => NumNominative => it- tlettax-il
s . NOrd => NumAdjectival => it- tlettax-il
n . NumPl
4 msec
> s . NCard => NumNominative => erbatax
s . NCard => NumAdjectival => erbatax-il
s . NOrd => NumNominative => l- erbatax-il
s . NOrd => NumAdjectival => l- erbatax-il
n . NumPl
8 msec
> s . NCard => NumNominative => dsatax
s . NCard => NumAdjectival => dsatax-il
s . NOrd => NumNominative => id- dsatax-il
s . NOrd => NumAdjectival => id- dsatax-il
n . NumPl
4 msec
> s . NCard => NumNominative => għoxrin
s . NCard => NumAdjectival => għoxrin
s . NOrd => NumNominative => l- għoxrin
s . NOrd => NumAdjectival => l- għoxrin
n . NumPl
4 msec
> s . NCard => NumNominative => wieħed u għoxrin
s . NCard => NumAdjectival => wieħed u għoxrin
s . NOrd => NumNominative => il- wieħed u għoxrin
s . NOrd => NumAdjectival => il- wieħed u għoxrin
n . NumPl
8 msec
> s . NCard => NumNominative => tnejn u tletin
s . NCard => NumAdjectival => tnejn u tletin
s . NOrd => NumNominative => it- tnejn u tletin
s . NOrd => NumAdjectival => it- tnejn u tletin
n . NumPl
4 msec
> s . NCard => NumNominative => ħamsa u ħamsin
s . NCard => NumAdjectival => ħamsa u ħamsin
s . NOrd => NumNominative => il- ħamsa u ħamsin
s . NOrd => NumAdjectival => il- ħamsa u ħamsin
n . NumPl
8 msec
> s . NCard => NumNominative => sebgħin
s . NCard => NumAdjectival => sebgħin
s . NOrd => NumNominative => is- sebgħin
s . NOrd => NumAdjectival => is- sebgħin
n . NumPl
0 msec
> s . NCard => NumNominative => disgħa u disgħin
s . NCard => NumAdjectival => disgħa u disgħin
s . NOrd => NumNominative => id- disgħa u disgħin
s . NOrd => NumAdjectival => id- disgħa u disgħin
n . NumPl
4 msec
> s . NCard => NumNominative => mija
s . NCard => NumAdjectival => mija
s . NOrd => NumNominative => il- mitt
s . NOrd => NumAdjectival => il- mitt
n . NumPl
4 msec
> s . NCard => NumNominative => mija u wieħed
s . NCard => NumAdjectival => mija u wieħed
s . NOrd => NumNominative => il- mija u wieħed
s . NOrd => NumAdjectival => il- mija u wieħed
n . NumPl
8 msec
> s . NCard => NumNominative => mija u tnejn
s . NCard => NumAdjectival => mija u żewġ
s . NOrd => NumNominative => il- mija u tnejn
s . NOrd => NumAdjectival => il- mija u żewġ
n . NumPl
4 msec
> s . NCard => NumNominative => mija u disgħa
s . NCard => NumAdjectival => mija u disa'
s . NOrd => NumNominative => il- mija u disgħa
s . NOrd => NumAdjectival => il- mija u disa'
n . NumPl
8 msec
> s . NCard => NumNominative => mija u għaxra
s . NCard => NumAdjectival => mija u għaxar
s . NOrd => NumNominative => il- mija u għaxra
s . NOrd => NumAdjectival => il- mija u għaxar
n . NumPl
4 msec
> s . NCard => NumNominative => mija u ħdax
s . NCard => NumAdjectival => mija u ħdax-il
s . NOrd => NumNominative => il- mija u ħdax
s . NOrd => NumAdjectival => il- mija u ħdax-il
n . NumPl
4 msec
> s . NCard => NumNominative => mija u tnax
s . NCard => NumAdjectival => mija u tnax-il
s . NOrd => NumNominative => il- mija u tnax
s . NOrd => NumAdjectival => il- mija u tnax-il
n . NumPl
8 msec
> s . NCard => NumNominative => mija u erbatax
s . NCard => NumAdjectival => mija u erbatax-il
s . NOrd => NumNominative => il- mija u erbatax
s . NOrd => NumAdjectival => il- mija u erbatax-il
n . NumPl
8 msec
> s . NCard => NumNominative => erba' mija
s . NCard => NumAdjectival => erba' mitt
s . NOrd => NumNominative => l- erba' mitt
s . NOrd => NumAdjectival => l- erba' mitt
n . NumPl
4 msec
> s . NCard => NumNominative => erba' mija u għaxra
s . NCard => NumAdjectival => erba' mija u għaxar
s . NOrd => NumNominative => l- erba' mija u għaxra
s . NOrd => NumAdjectival => l- erba' mija u għaxar
n . NumPl
8 msec
> s . NCard => NumNominative => ħames mija u wieħed
s . NCard => NumAdjectival => ħames mija u wieħed
s . NOrd => NumNominative => il- ħames mija u wieħed
s . NOrd => NumAdjectival => il- ħames mija u wieħed
n . NumPl
12 msec
> s . NCard => NumNominative => sitt mija
s . NCard => NumAdjectival => sitt mitt
s . NOrd => NumNominative => is- sitt mitt
s . NOrd => NumAdjectival => is- sitt mitt
n . NumPl
4 msec
> s . NCard => NumNominative => tmin mija
s . NCard => NumAdjectival => tmin mitt
s . NOrd => NumNominative => it- tmin mitt
s . NOrd => NumAdjectival => it- tmin mitt
n . NumPl
4 msec
> s . NCard => NumNominative => disa' mija u ħdax
s . NCard => NumAdjectival => disa' mija u ħdax-il
s . NOrd => NumNominative => id- disa' mija u ħdax
s . NOrd => NumAdjectival => id- disa' mija u ħdax-il
n . NumPl
8 msec
> s . NCard => NumNominative => elf
s . NCard => NumAdjectival => elf
s . NOrd => NumNominative => l- elf
s . NOrd => NumAdjectival => l- elf
n . NumPl
4 msec
> s . NCard => NumNominative => ħdax elf
s . NCard => NumAdjectival => ħdax elf
s . NOrd => NumNominative => il- ħdax-il elf
s . NOrd => NumAdjectival => il- ħdax-il elf
n . NumPl
4 msec
> s . NCard => NumNominative => ħdax elf u tnax
s . NCard => NumAdjectival => ħdax elf u tnax
s . NOrd => NumNominative => il- ħdax-il elf u tnax
s . NOrd => NumAdjectival => il- ħdax-il elf u tnax
n . NumPl
8 msec
> s . NCard => NumNominative => erbgħin elf
s . NCard => NumAdjectival => erbgħin elf
s . NOrd => NumNominative => l- erbgħin elf
s . NOrd => NumAdjectival => l- erbgħin elf
n . NumPl
8 msec
> s . NCard => NumNominative => wieħed u disgħin elf u seba' mija
s . NCard => NumAdjectival => wieħed u disgħin elf u seba' mija
s . NOrd => NumNominative => il- wieħed u disgħin elf u seba' mija
s . NOrd => NumAdjectival => il- wieħed u disgħin elf u seba' mija
n . NumPl
8 msec
> s . NCard => NumNominative => mija elf
s . NCard => NumAdjectival => mija elf
s . NOrd => NumNominative => il- mitt elf
s . NOrd => NumAdjectival => il- mitt elf
n . NumPl
4 msec
> s . NCard => NumNominative => mija elf u mija
s . NCard => NumAdjectival => mija elf u mija
s . NOrd => NumNominative => il- mitt elf u mija
s . NOrd => NumAdjectival => il- mitt elf u mija
n . NumPl
4 msec
> s . NCard => NumNominative => mija elf u wieħed u tletin
s . NCard => NumAdjectival => mija elf u wieħed u tletin
s . NOrd => NumNominative => il- mitt elf u wieħed u tletin
s . NOrd => NumAdjectival => il- mitt elf u wieħed u tletin
n . NumPl
8 msec
> s . NCard => NumNominative => mija u għaxra elf
s . NCard => NumAdjectival => mija u għaxra elf
s . NOrd => NumNominative => il- mija u għaxra elf
s . NOrd => NumAdjectival => il- mija u għaxra elf
n . NumPl
4 msec
> s . NCard => NumNominative => mija u ħdax elf
s . NCard => NumAdjectival => mija u ħdax elf
s . NOrd => NumNominative => il- mija u ħdax elf
s . NOrd => NumAdjectival => il- mija u ħdax elf
n . NumPl
8 msec
> s . NCard => NumNominative => mija u wieħed u għoxrin elf u mija
s . NCard => NumAdjectival => mija u wieħed u għoxrin elf u mija
s . NOrd => NumNominative => il- mija u wieħed u għoxrin elf u mija
s . NOrd => NumAdjectival => il- mija u wieħed u għoxrin elf u mija
n . NumPl
8 msec
> s . NCard => NumNominative => mija u tletin elf
s . NCard => NumAdjectival => mija u tletin elf
s . NOrd => NumNominative => il- mija u tletin elf
s . NOrd => NumAdjectival => il- mija u tletin elf
n . NumPl
4 msec
> s . NCard => NumNominative => mija u wieħed u erbgħin elf
s . NCard => NumAdjectival => mija u wieħed u erbgħin elf
s . NOrd => NumNominative => il- mija u wieħed u erbgħin elf
s . NOrd => NumAdjectival => il- mija u wieħed u erbgħin elf
n . NumPl
8 msec
> s . NCard => NumNominative => mija u sittin elf
s . NCard => NumAdjectival => mija u sittin elf
s . NOrd => NumNominative => il- mija u sittin elf
s . NOrd => NumAdjectival => il- mija u sittin elf
n . NumPl
12 msec
> s . NCard => NumNominative => mitejn elf
s . NCard => NumAdjectival => mitejn elf
s . NOrd => NumNominative => il- mitejn elf
s . NOrd => NumAdjectival => il- mitejn elf
n . NumPl
4 msec
> s . NCard => NumNominative => mitejn elf u erba' mija u għaxra
s . NCard => NumAdjectival => mitejn elf u erba' mija u għaxra
s . NOrd => NumNominative => il- mitejn elf u erba' mija u għaxra
s . NOrd => NumAdjectival => il- mitejn elf u erba' mija u għaxra
n . NumPl
8 msec
> s . NCard => NumNominative => sitt mija elf
s . NCard => NumAdjectival => sitt mija elf
s . NOrd => NumNominative => is- sitt mitt elf
s . NOrd => NumAdjectival => is- sitt mitt elf
n . NumPl
8 msec
> s . NCard => NumNominative => sitt mija u għaxra elf u għaxra
s . NCard => NumAdjectival => sitt mija u għaxra elf u għaxra
s . NOrd => NumNominative => is- sitt mija u għaxra elf u għaxra
s . NOrd => NumAdjectival => is- sitt mija u għaxra elf u għaxra
n . NumPl
4 msec
> s . NCard => NumNominative => seba' mija elf u sitt mija u għaxra
s . NCard => NumAdjectival => seba' mija elf u sitt mija u għaxra
s . NOrd => NumNominative => is- seba' mitt elf u sitt mija u għaxra
s . NOrd => NumAdjectival => is- seba' mitt elf u sitt mija u għaxra
n . NumPl
12 msec
> s . NCard => NumNominative => tmin mija u ħamsin elf u sbatax
s . NCard => NumAdjectival => tmin mija u ħamsin elf u sbatax
s . NOrd => NumNominative => it- tmin mija u ħamsin elf u sbatax
s . NOrd => NumAdjectival => it- tmin mija u ħamsin elf u sbatax
n . NumPl
12 msec
> s . NCard => NumNominative => disa' mija elf u mija u għaxra
s . NCard => NumAdjectival => disa' mija elf u mija u għaxra
s . NOrd => NumNominative => id- disa' mitt elf u mija u għaxra
s . NOrd => NumAdjectival => id- disa' mitt elf u mija u għaxra
n . NumPl
8 msec
> s . NCard => NumNominative => disa' mija elf u mitejn
s . NCard => NumAdjectival => disa' mija elf u mitejn
s . NOrd => NumNominative => id- disa' mitt elf u mitejn
s . NOrd => NumAdjectival => id- disa' mitt elf u mitejn
n . NumPl
8 msec
> See you.
